qemu/tests/virtio_driver_sign_check.py

```diff
@@ -48,12 +48,22 @@ def run(test, params, env):
     timeout = float(params.get("login_timeout", 240))
     session = vm.wait_for_login(timeout=timeout)
     signtool_cmd = params["signtool_cmd"]
+    drv_name = params["tested_driver"].lower()
     list_files_cmd = 'dir /s /b %s | find /i "%s" | find "%s"'
+    if "prewhql" in params["cdrom_virtio"]:
+        v_mark = "/pa"
```

Comment:
I think we can define a parameter in cfg and get the value here. It might be more reasonable. What do you think about it?

Reply:
updated, thanks~

Reply:
Seems there is no verify_mark config in .cfg file.

Reply:
Hi xiagao, as I discussed with leidong, we get the conclusion for set default value for verify_mark param in py file.
v_mark = params.get("verify_mark", "/kp")

Reply:
updated as we discussed, thanks all~

Reply:
I would suggest update `verify_mark` where you define `cdrom_virtio` with the `prewhql` keyword. Current implementation looks like a hard-code for me.

Reply:
Hi @vivianQizhu Updated the code, thanks~